reference/scalability_guide/mongo_cursor_timeout_exception.rst

```diff
@@ -0,0 +1,21 @@
+How to get rid of the MongoCursorTimeoutException
+-------------------------------------------------
+
+In the PIM we always iterate over cursors and paginate them to avoid big queries on the MongoDB database. But for some low level operations which are not paginable you could encounter the following exception:
```

Comment:
"But for some low level operations which are not paginable you could encounter the following exception"

Would maybe add "with a large volume of data" and I would give the real example, aka, get the indexes